CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this
 project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.3.0] - 2020-05-19
```

Comment:
FYI changelog is now autogenerated, so you don't need to update this manually.

Reply:
ah I didn't know that. Thanks.